lib/ioki/model/platform/captcha.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Platform
+      class Captcha < Base
+        attribute :type, on: :read, type: :string
+        attribute :id, on: :read, type: :string
+        attribute :created_at, on: :read, type: :date_time
+        attribute :updated_at, on: :read, type: :date_time
+        attribute :captcha_type, on: :read, type: :string
+        attribute :question_prompt, on: :read, type: :string
+        attribute :image_url, on: :read, type: :string
+
```

Comment:
🤓 Nitpick, but what's this blank line doing? :D

Reply:
It is feng shui of coding. It let's the energy flow freely through the code :cowboy_hat_face: 

In seriousness, all the top attributes are `on: :read` and the last one is `on: :create`, so this visually groups them. I usually like to have some grouping in such lists for better understanding, but not sure if this is the right one in this case.

Reply:
Neat, didn't see that. Makes sense.